src/components/modules/shared/ArticleCountOptOut.tsx

```diff
@@ -41,16 +45,15 @@ const overlayContainer = (type: ArticleCountOptOutType): SerializedStyles => css
     z-index: 100;
     left: ${space[4]}px;
     right: ${space[4]}px;
-    ${type === 'banner' ? 'bottom: 21px;' : ''}
+    ${isBanner(type) ? 'bottom: 21px;' : ''}
 
     ${from.tablet} {
-        width: 325px;
+        width: 400px;
         left: 0;
+        ${isBanner(type) ? 'bottom: -90px;' : ''}
```

Comment:
Not ideal but it works for now. We need to review the UX + implementation of the popup in the banner